configuration.md

```diff
@@ -403,6 +404,33 @@ future, to accommodate clients with incorrect clocks.
 
 Default: 1 day
 
+#### raw_database conflict_resolver
+
+When a metric gets written more than one time at the same exact millisecond
+offset you have a conflict we have to resolve. All operations in IRONdb are
+commutative and this lets us avoid complicated consensus algorithms for data.
+Conflicts, therefore, need to choose a winner and this choice needs to be
+consistent across the cluster. IRONdb gives you the following choices for
+conflict resolution should a datapoint appear more than once at the same
+millisecond.
+
+* `abs_biggest` - save the largest by absolute value.
+* `last_abs_biggest` - if used with the [IRONdb-relay](irondb-relay.md)
```

Comment:
What happens if I set this but am not ingesting via irondb-relay?  Should try to be clear about whether these should be used at all without the relay.

Reply:
It's fine to set and not use the relay... generation counters would always be zero (unless/until we come up with a way to let people overwrite generation outside of relay usage)